src/seedu/addressbook/parser/Parser.java

```diff
@@ -50,7 +50,7 @@ public Parser() {}
      * @param userInput full user input string
      * @return the command based on the user input
      */
-    public Command parseCommand(String userInput) {
+    public static Command parseCommand(String userInput) {
```

Comment:
are there places where you need to update these references since they have become static already?